### PR TITLE
[EWL-3570] : creates EC search bar component

### PR DESCRIPTION
**Pull request by rpama**
## JIRA Ticket(s)
- [EWL-3570: EC | Design Search bar component](https://issues.ama-assn.org/browse/EWL-3570)

## Description:
Creates a component for the EC search bar.

## To Test:
- [x] Go to Organisms > EC > Search Bar
- [x] See that there is a component there like the wireframe

## Automated Test:
N/A

## Relevant Screenshots/GIFs:
None.

## Remaining Tasks:
Probably this gets fleshed out more when more details about the requirements are provided.

## Additional Notes:
No.

---

[Pull Request Guidelines](https://github.com/palantirnet/development_documentation/blob/master/guidelines/pr_review.md)

### CODE REVIEW

**Comment by rpama** [comment]
Marking as [WIP] because I have to update the mobile to hide the Library button.

**Comment by rpama** [comment]
Mobile style in which library button is not displayed has been added. Considering this dev complete again and ready for review.

File: styleguide/source/_patterns/02-organisms/05-ec/01-search-bar/_search-bar.scss [hunk 1]
```diff
@@ -0,0 +1,62 @@
+.search-bar {
```
Comment (deadarm) [inline styleguide/source/_patterns/02-organisms/05-ec/01-search-bar/_search-bar.scss]: Suggestion to add 
```
/**
 * @file search-bar.scss
 *
 */
```
to top of file

File: styleguide/source/_patterns/02-organisms/05-ec/01-search-bar/search-bar.twig [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<div class="search-bar">
+  <div class="search-bar-inner">
+    <div class="search-bar_library">
+      {% include "atoms-button-link-secondary" with {'content': 'Library'} %}
+    </div>
+    <div class="search-bar_form">
+      {% include "molecules-search-field" with {'placeholder': 'Search Courses'} %}
+    </div>
+  </div>
+</div>
```
Comment (deadarm) [inline styleguide/source/_patterns/02-organisms/05-ec/01-search-bar/search-bar.twig]: Suggestion to add
```
{#
 # @file search-bar.twig
 #}
```
to bottom of twig file.

**Comment by mrspete** [comment]
@rpama I agree that we should rename this component to be more specific to the ed center.

File: styleguide/source/_patterns/02-organisms/05-ec/01-search-bar/_search-bar.scss (hunk 1 above)
Comment (deadarm) [inline styleguide/source/_patterns/02-organisms/05-ec/01-search-bar/_search-bar.scss]: This change no longer needed.

File: styleguide/source/_patterns/02-organisms/05-ec/01-search-bar/search-bar.twig (hunk 2 above)
Comment (deadarm) [inline styleguide/source/_patterns/02-organisms/05-ec/01-search-bar/search-bar.twig]: This change no longer needed.

**Comment by deadarm** [comment]
LGTM. Thanks @rpama !

**Pull request merged**